### PR TITLE
Update CODEOWNERS

### PR DESCRIPTION
**Pull request by y0urself**
**What**:

**Why**:

**How**:

**Checklist**:

- [ ] Tests
- [ ] PR merge commit message adjusted

### CODE REVIEW

**Review by bjoernricks — DISMISSED** [review]

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1,2 +1,6 @@
 # default reviewers
 *                 @greenbone/openvas-scanner-maintainers
+
+# devops
+.github/*         @greenbone/devops
+.docker/*         @greenbone/devops
```
Comment (ArnoStiefvater) [inline .github/CODEOWNERS:6]: In https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners there is the example:

```
# The `docs/*` pattern will match files like
# `docs/getting-started.md` but not further nested files like
# `docs/build-app/troubleshooting.md`.
docs/*  docs@example.com
```
You probably want `.github/`. Although there is no example in the doc about hidden folders.

Comment (y0urself) [reply .github/CODEOWNERS:6]: n4rf.

Comment (y0urself) [inline .github/CODEOWNERS:6]:
```suggestion
.github/         @greenbone/devops
.docker/         @greenbone/devops
```

**Review by ArnoStiefvater — DISMISSED** [review]

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -1,2 +1,6 @@
 # default reviewers
 *                 @greenbone/openvas-scanner-maintainers
+
+# devops
+.github/         @greenbone/devops
+.docker/         @greenbone/devops
```
Comment (y0urself) [inline .github/CODEOWNERS:6]:
```suggestion
.github/          @greenbone/devops
.docker/          @greenbone/devops
```

**Review by ArnoStiefvater — APPROVED** [review]

**Review by ArnoStiefvater — APPROVED** [review]

**Review by Kraemii — APPROVED** [review]

**Pull request merged**